numba_dpex/experimental/_kernel_dpcpp_spirv_overloads/_atomic_ref_overloads.py

```diff
@@ -538,11 +538,12 @@ def _intrinsic_compare_exchange_gen(context, builder, sig, args):
 def _check_if_supported_ref(ref):
     supported = True
 
-    if not isinstance(ref, USMNdArray):
+    if not isinstance(ref, (USMNdArray)):
```

Comment:
Why is it wrapped in tuple?

Reply:
Leftover code. At one point I had a tuple as the argument to the isinstance call something like: `isinstance(ref, (USMNdArray, LocalAccessorType))`. I missed removing the parenthesis after removing `LocalAccessorType` since every `LocalAccessorType` is also an `USMNdArray`.